pilot/helpers/Project.py

```diff
@@ -69,6 +70,11 @@ def __init__(self, args, name=None, description=None, user_stories=None, user_ta
             self.architecture = architecture
         # if development_plan is not None:
         #     self.development_plan = development_plan
+        self.dot_pilot_gpt = DotGptPilot(log_chat_completions=enable_dot_pilot_gpt)
```

Comment:
can we do this "if  enable_dot_pilot_gpt" ? Just to make sure it doesn't affect regular user at all